client/src/components/StepsQuestions/UploadImg.js

```diff
@@ -0,0 +1,106 @@
+import React, { Component } from 'react';
+import { Upload, Icon, message } from 'antd';
+import propTypes from 'prop-types';
+
+import './index.css';
+
+const getBase64 = (img, callback) => {
+  const reader = new FileReader();
+  reader.addEventListener('load', () => callback(reader.result));
+  reader.readAsDataURL(img);
+};
+
+const beforeUpload = file => {
+  const isJpgOrPng = file.type === 'image/jpeg' || file.type === 'image/png';
+
+  if (!isJpgOrPng) {
+    message.error('you can only upload JPG/PNG file!');
+  }
+  const isLt2M = file.size / 1024 / 1024 < 2;
+  if (!isLt2M) {
+    message.error('Image must smaller than 2MB');
+  }
+
+  return isJpgOrPng && isLt2M;
+};
+
+class UploadImg extends Component {
+  state = {
+    loading: false,
+  };
+
+  handleChange = info => {
+    if (info.file.status === 'uploading') {
+      this.setState({ loading: true });
+      return;
+    }
+    if (info.file.status === 'done') {
+      // Get this url from response in real world.
+      getBase64(info.file.originFileObj, imgUrl => {
+        const { handelChange, current } = this.props;
+        if (current === 5) {
+          handelChange(imgUrl, 'imgUrlOne');
+          return this.setState({
+            loading: false,
+          });
+        }
+        if (current === 6) {
+          handelChange(imgUrl, 'imgUrlTwo');
+          return this.setState({
+            loading: false,
+          });
+        }
+
+        return '';
+      });
+    }
+  };
+
+  render() {
+    const { loading } = this.state;
+    const { values, current } = this.props;
+    let imgUrl;
+    if (current === 5) {
```

Comment:
The `current === 5` and `current === 6` checks appear twice (here, and line 41 / 47) and it's not clear what they mean, so it might make this code easier to understand if you created variables at the top of the file and gave them a descriptive name:
```
const FIRST_IMAGE_UPLOAD_STEP = 5
const SECOND_IMAGE_UPLOAD_STEP = 6
```
and then used those variables in these if-statements:
`if ( current === FIRST_IMAGE_UPLOAD_STEP)`